content/ec2-auto-scaling-with-multiple-instance-types-and-purchase-options/create_asg.md

```diff
@@ -37,4 +44,86 @@ This command will not return any output if it is successful.
 
 # Optional exercise
 
-Now that you have deployed an EC2 Auto Scaling group with Mixed Instance Types and Purchase options, take some time to go through the different configurations in the [console](https://console.aws.amazon.com/ec2autoscaling/home?#/). Click on the **myEC2Workshop** Auto Scaling group and go to the *Purchase options and instance types* section and try to edit the instance types configured on the Auto Scaling group and change the "primary instance type" to see how the Auto Scaling console provides you a recommended list of instance types based on your selected instance type.
\ No newline at end of file
+Now that you have deployed an EC2 Auto Scaling group with Mixed Instance Types and Purchase options, take some time to go through the different configurations in the [console](https://console.aws.amazon.com/ec2autoscaling/home?#/). Click on the **myEC2Workshop** Auto Scaling group and go to the *Instance type requirements* section and click *Edit* button and scroll down to *Preview matching instance types*. Expand to see list of instance types based on your selected instance attributes. Try to edit the instance requirements configured on the Auto Scaling group and see how this preview list changes.
```

Comment:
This section will cause confusion and it's also error prone as in previous section you have already created ASG with name myEC2Workshop and ABIS. Now in this section you are creating another ASG with manually selecting instances. Let's discuss we can keep only ABIS.